server/conf/evolutions/default/66.sql

```diff
@@ -0,0 +1,66 @@
+# --- !Ups
+--Drop the existing view
+DROP MATERIALIZED VIEW IF EXISTS monthly_submissions_reporting_view;
+-- create a new view
```

Comment:
**nitpick:** Can update capitalization to be consistent with above.

Reply:
I am planning to remove the comments and other evolutions do not have it. These comments were added for my understanding.